flink-python/src/main/java/org/apache/flink/api/common/python/PythonBridgeUtils.java

```diff
@@ -74,6 +75,20 @@
 		return unpickledData;
 	}
 
+	public static List<?> readPythonObjects(String fileName) throws IOException {
```

Comment:
This util method will also be used in StreamTableEnvironment._from_elements(), have you tested it？Maybe we can keep it here, and create a new method for reading DataStream element objects.